PeoplePicker.html

```diff
@@ -1,230 +1,183 @@
-    authenticator
-        .authenticate(OfficeHelpers.DefaultEndpoints.Microsoft, true)
-        .then(function (token) {  
-          
-        })
-        .catch(OfficeHelpers.Utilities.log);
-  });
-
-
-
-</script>
-
-
-<script>
-  
-var userArray = new Array();
-var personaImageBase = "PersonaImage";
-var nameElementBase = "PersonaName";
-var emailElementBase = "PersonaEmail";
-var personaElementBase = "Persona";
-
-function getUsers(){
-
-  // Get the access token and create a Microsoft Graph client
-  var authObject = authenticator.tokens.get('Microsoft');
-  var accessToken = authObject.access_token;
-  const client = MicrosoftGraph.Client.init({
-    authProvider: (done) => {
-        done(null, accessToken); //first parameter takes an error if you can't get an access token
-    }
-    });
-
-    client
-    .api('me/people')
-    .version("beta")
-    .top(4)
-    .get()
-    .then(function(res){
-
-      // Populate the text fields.
-        var counter = 0;
-        res.value.forEach(function(userObject){
-        var nameElement = nameElementBase.concat(counter);
-        var emailElement = emailElementBase.concat(counter);
-        var personaElement = personaElementBase.concat(counter);
-        var displayName = userObject.displayName;
-        var emailAddress = userObject.userPrincipalName;
-        var userId = userObject.id;
-        userArray.push(userId);
-        document.getElementById(nameElement).innerHTML = displayName;
-        document.getElementById(emailElement).innerHTML = emailAddress;
-        counter++;
-        });
-    })
-    .then(function(){
-
-      // Retrieve user photos and populate the image fields.
-      for (var i =0; i < userArray.length; i++){
-        (function(userId) {        
-          var personaImage = personaImageBase.concat(i);
-          client
-          .api('/users/' + userId + '/photo/$value')
-          .responseType('blob')
-          .header("Cache-Control", "no-cache")
-          .get((err, res, rawResponse) => {
-            const url = window.URL;
-            const blobUrl = url.createObjectURL(rawResponse.xhr.response);
-            document.getElementById(personaImage).setAttribute("src", blobUrl);
-          });
-        })(userArray[i]);
-      }
-    })
-    .catch((err) => {
-        console.log(err);
-    });
-  
-}
-
-
-</script>
-
-  </body>
-</html>
\ No newline at end of file
+          </div>        
+          <!-- Create a People Picker from the HTML elements -->
+          <script type="text/javascript">
+            var PeoplePickerElements = document.querySelectorAll(".ms-PeoplePicker");
+            for (var i = 0; i < PeoplePickerElements.length; i++) {
+              new fabric['PeoplePicker'](PeoplePickerElements[i]);
+            }
+          </script>       
+        
+          <!-- Authenticate with Office JS Helper -->
+          <script>
+            var clientId = "ae0b221d-6c54-4358-a365-6fb17d497305";
```

Comment:
Client id could be abstracted in a secrets.js file where you feed the client ID to a window variable called window.ClientId. and reference it in line 114 as 

`var clientId = window.ClientId;`

Provide a secrets.sample.js file in the code with an empty value in the window.ClientId variable and offer documentation that states that the user should save this file as secrets.js and input the client id received on app registration. 

This helps with security